theano/sandbox/cuda/tests/test_conv_cuda_ndarray.py

```diff
@@ -845,10 +845,15 @@ def conv_grad(mode, bs, ch, nf, rImg1, rImg2, rFlt1, rFlt2, subsample, op):
     outputs.extend([corr_op_dik, conv_op_dik,
                     corr_op_dki, conv_op_dki])
 
-    # TODO: fix when the abstractconv tests can pass debug mode.
-    mode = theano_mode
-    if theano.config.mode == 'DEBUG_MODE':
```

Comment:
Without CuDNN, the test takes:
- 200 s in default mode
- 2000 s in DEBUG_MODE

This is probably the same time it was taking before we introduced the abstract convolution, but I'm OK with not putting back debug mode before the release.